packages/connect-core/src/utils/app.ts

```diff
@@ -1,11 +1,32 @@
 import { utils as ethersUtils } from 'ethers'
 
-import { AppMethod } from '../types'
+import { Abi, AppMethod } from '../types'
 import App from '../entities/App'
 
 export const apmAppId = (appName: string): string =>
   ethersUtils.namehash(`${appName}.aragonpm.eth`)
 
+const checkSignature = (signature: string, abi: Abi): string => {
```

Comment:
I would expect `checkSignature()` to return a boolean, should it be something like `signatureFromAbi()` instead? :thinking: 

Also we should use `function checkSignature() {` to follow the coding style conventions.